frontend/src/views/reports/AuditReporting.vue

```diff
@@ -130,7 +130,31 @@ export default {
       firstRecordIndex: 0,
       loading: false,
       lazyParams: {},
-      transactionOptions: ['CheckEligibility', 'PHNInquiry', 'PHNLookup', 'EnrollSubscriber', 'GetPersonDetails', 'NameSearch', 'AddGroupMember', 'AddDependent', 'UpdateNumberAndDept', 'CancelDependent', 'ContractInquiry', 'GetContractAddress', 'UpdateContractAddress'],
+      transactionOptions: [
+        'AddDependent',
+        'AddGroupMember',
+        'CancelDependent',
+        'CancelGroupMember',
+        'ChangeCancelDate',
+        'ChangeEffectiveDate',
+        'CheckEligibility',
+        'ContractInquiry',
+        'EnrollSubscriber',
+        'ExtendCancelDate',
```

Comment:
ExtendCancelDate has not been built . Can it be removed?

Reply:
R51 (Extend Cancel Date) will likely be added back in.